src/styles/blocks/header.scss

```
@@ -0,0 +1,58 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px #0000000d;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    justify-content: center;
+    text-transform: uppercase;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    display: flex;
+    padding-left: 20px;
+    line-height: 60px;
+
+    &:first-child {
+      padding-left: 0;
+    }
+  }
+
+}
+
+.link {
```

should be as an element of nav block
```suggestion
.nav__link {
```